pytest.txt

```diff
@@ -0,0 +1,7 @@
+pytest~=6.2.5
```

Comment:
No need to add this file. Pls use the file with same name under the `tools/install_deps`